include/rtc_logging.h

```diff
@@ -0,0 +1,25 @@
+#ifndef LIB_WEBRTC_RTC_LOGGING_HXX
+#define LIB_WEBRTC_RTC_LOGGING_HXX
+
+#include "rtc_types.h"
+
+namespace libwebrtc {
+
+  enum RTCLoggingSeverity {
+    Verbose,
+    Info,
+    Warning,
+    Error,
+    None,
+  };
+
+  typedef void (*RTCCallbackLoggerMessageHandler)(const std::string& message);
```

Comment:
Avoid using std::string to ensure dynamic library portability, as this may cause crashes due to inconsistent libstdc++ versions in the environment.
```suggestion
  typedef void (*RTCCallbackLoggerMessageHandler)(const string& message);
```